package-examples/helloworld-set/README.md

```diff
@@ -0,0 +1,75 @@
+# helloworld-set
+
+This is a simple package that showcases a kpt package that has a number of
+setters in it.
+
+## Steps
+
+This is a sample workflow where 
+
+1. [Fetch the package](#fetch-the-package)
+2. [View the package contents](#view-the-package-contents)
+3. [Configure functions](#configure-functions)
+4. [Render the declared values](#render-the-declared-values)
+5. [Apply the package](#apply-the-package)
+
+### Fetch the package
+
+Get the example package on to local using `kpt pkg get`
+
+  $ kpt pkg get https://github.com/GoogleContainerTools/kpt.git/package-examples/helloworld-set
+
+    fetching package /package-examples/helloworld-set from https://github.com/GoogleContainerTools/kpt to helloworld-set
+
+### View the package contents
+
+List the package contents in a tree structure.
+
+  $ kpt pkg tree helloworld-set/
+
+    PKG: helloworld-set
+    ├── [Kptfile]  Kptfile helloworld-set
+    ├── [deploy.yaml]  Deployment helloworld-gke
+    └── [service.yaml]  Service helloworld-gke
+
+### Configure functions
+
+The package contains a function pipeline in the `Kptfile` which has
+one `apply-setter` function.  The `apply-setter` function allows you to
+set a simple value throughout the package configuration.  In this case
+you can set the replicas, image, tag and http port of a simple application.
+
+  pipeline:
+    mutators:
+      - image: gcr.io/kpt-fn/apply-setters:unstable
```

Comment:
Use v0. Not available yet, but will be released soon.

Reply:
I'd rather not block on another release to re-enable the testing.  I think search and replace of `:unstable` is going to be a simple PR.